packages/components/login-widget/src/providers.ts

```diff
@@ -49,6 +49,7 @@ export default function getAvailableWallets() {
           icon: IconMetamaskWallet,
           isInstalled: !isSSR && Boolean(window.ethereum),
           connect: () =>
+            // @ts-ignore
```

Comment:
Is this the cause of your build failure? Since we had the definitions for the providers here, there should not need the use of `ts-ignore`:

https://github.com/consolelabs/websites/blob/b039992a7c504bc058e6e648781180f58b44e949/packages/components/login-widget/provider.d.ts#L1-L9

Reply:
It's saying the property could be undefined so maybe we can fix it by a condition check or optional chaining?

<img width="1023" alt="image" src="https://github.com/consolelabs/websites/assets/12707960/d2681541-6749-4698-9724-d7234b44459a">